vignettes/laminr.Rmd

```diff
@@ -55,7 +55,6 @@ Create your default database `db` object for this R session:
 
 ```{r connect-default}
 db <- connect()
-db
```

Comment:
I removed it because it confusingly displays `cellxgene` -- see discussion in other PR.